### PR TITLE
cli: Refactor js cli to support all contracts + add improvements

### PR DESCRIPTION
**Pull request by kcsongor**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/1100)

### CODE REVIEW

**Review by bruce-riley — APPROVED** [review]

**Comment by bruce-riley** [comment]
Oh, and I forgot to mention, thank you so much for doing this! It will make my life much easier!

File: clients/js/networks.ts [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+function get_env_var(env: string): string {
+    const v = process.env[env]
+    if (v === undefined) {
+        throw new Error(`Env variable ${env} undefined`)
+    }
+    return v
+}
+
+const mainnet = {
```
[the remark this answers is not in the document]
Comment (kcsongor) [reply clients/js/networks.ts:49]: Good point - we should really have this stuff in the sdk! Then there's a single source of truth, and both the bridge ui and the cli can use it. I'll open a PR to do it

File: clients/js/evm.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import { networks } from "./networks"
+import { impossible, Payload } from "./vaa"
+
+export async function execute_governance_evm(
+    payload: Payload,
+    vaa: Buffer,
+    network: typeof networks.mainnet.ethereum
+) {
+    let provider = new ethers.providers.JsonRpcProvider(network.rpc)
+    let signer = new ethers.Wallet(network.key, provider)
```
[the remark this answers is not in the document]
Comment (kcsongor) [reply clients/js/evm.ts:12]: sounds like this function should just take the chain id directly then 🤔

**Review by bruce-riley — DISMISSED** [review]

**Review by evan-gray — CHANGES_REQUESTED** [review]
seems safe given that it's all in clients, and I definitely approve of it all being consolidated. can we support "DEVNET" as a network as well.

**Review by evan-gray — APPROVED** [review]

**Review by bruce-riley — APPROVED** [review]

**Pull request merged**